mx.sulong/mx_sulong.py

```diff
@@ -1155,6 +1175,8 @@ def checkNoHttp(args=None):
     'su-travis-sulong' : [travisTestSulong, ''],
     'su-travis-jruby' : [travisJRuby, ''],
     'su-travis-argon2' : [travisArgon2, ''],
+    'su-ecj' : [runEcjTestCases, ''],
```

Comment:
I think we do not need a separate mx command for that. If you are using `javac` per default you can use `mx build --jdt <path>` to compile with the Eclipse compiler and if you are using the Eclipse compiler per default (by having set the `JDT` environment variable) you can use `mx build --force-javac`.